src/registry.json

```diff
@@ -1592,6 +1592,35 @@
         }
       }
     },
+    "npm:@astar-network/snap": {
+      "id": "npm:@astar-network/snap",
+      "metadata": {
+        "name": "Astar Wallet",
+        "author": {
+          "name": "Astar Network",
+          "website": "https://astar.network/"
+        },
+        "website": "https://portal.astar.network/shibuya-testnet/assets",
```

Comment:
Does this intentionally link to a testnet?

Reply:
Yes

Reply:
Mainnet will be ready for the Snap in a few weeks